### PR TITLE
Blog app: add devise

### PR DESCRIPTION
**Pull request by RitaDee**
### In this PR, we implemented the following 👎🏼 

- Remove `current_user` method in `ApplicationController`, devise will provide us with one.
- [Install and setup devise](https://github.com/heartcombo/devise#getting-started).
  - Can register a new user.
  - User logs in with a combination of email and password.
  - Hashed password should be stored in the database.
  - Ask for confirmation of email.
  - Can reset password.

### CODE REVIEW

**Review by brytebee — APPROVED** [review]
Hi Team 👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/user.rb [hunk 1]
```diff
@@ -1,10 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (brytebee) [inline app/models/user.rb]: You should also add and implement the functionality of confirming the email of the users of your app.

image

**Comment by RitaDee** [comment]
Hello @brytebee,

Thank you for the review.

Please kindly state the changes required of me.

**Review by brytebee — CHANGES_REQUESTED** [review]

**Comment by brytebee** [comment]
@RitaDee and @Divinity-dev and @HanaSabih I think it was a mixup with the network and it was pending on my PC but it is fixed now and you guys can make the updates.

File: app/models/user.rb (hunk 1 above)
Comment (RitaDee) [reply app/models/user.rb]: Noted with so many thanks @brytebee

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi team 👋, 

Great job implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by RitaDee** [comment]
Thank you for the review @codepantha

**Pull request merged**